<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>function_getRotationMatrix2D</class>
 <widget class="QWidget" name="function_getRotationMatrix2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="9,1">
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>参数说明</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>center：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_x"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_y"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>angle：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_angle"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>scale：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_scale"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_arg_ok">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>输出结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="tb_result"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
